PaddleHQ/checkout-sdk-go
# PAY-1076 Rename imports to PaddleHQ

- Renamed all imports to PaddleHQ from checkout
- Updated go.mod to PaddleHQ

How to test
- Ensure there are no references to `github.com/checkout/checkout-sdk-go/` (documentation is fine)
